shell/packages/sandstorm-db/db.js

```diff
@@ -732,9 +733,12 @@ _.extend(SandstormDb.prototype, {
 
   getIdentity: function getIdentity (identityId) {
     check(identityId, String);
-    var user = Meteor.users.findOne({"identities.id": identityId}, {fields: {"identities.$": 1}});
+    var user = Meteor.users.findOne({"identities.id": identityId},
+                                    {fields: {"identities.$": 1, "services": 1}});
+    // TODO(cleanup): Minimongo doesn't support "$", so this actually doesn't work on the client.
```

Comment:
If that's the case, then consider moving these inside an `if (Meteor.isServer)` block?